docs/integrations/builtin/core-nodes/n8n-nodes-base.datatable/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Data table
+description: Documentation for the data table node in n8n, a workflow automation platform. Includes guidance on usage, and links to examples.
+contentType: [integration, reference]
+priority: critical
+tags:
+  - data table node
+  - data
+hide:
+  - tags
+search:
+  boost: 1.5
+---
+
+# Data table
+
+The Data Table node allows you to permanently save data across workflow executions in a table format. It provides functionality to perform various data operations on stored data.
+
+## Node parameters
+
+### Resource
```

Comment:
Since this node has a single resource (rows), it might make sense to remove this section and mention that all of the operations apply to rows.